### PR TITLE
feat: add option to not check autos

### PR DESCRIPTION
**Pull request by steven-murray**
Adds an option not to check autos at all, since this takes ~1min for a simulation that takes <2min.

### CODE REVIEW

File: hera_sim/visibilities/cli.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def run_vis_sim(args):
         comm.Barrier()
         sys.exit(0)
 
-    if myid == 0:
+    if args.run_auto_check:
```
Comment (piyanatk) [inline hera_sim/visibilities/cli.py:170]: The auto check should still be done by rank 0 process, right?

**Review by piyanatk — APPROVED** [review]
I think the auto checking and compression should be done by rank 0 process, right? Otherwise, look good, Steven.

File: hera_sim/visibilities/cli.py (hunk 1 above)
Comment (steven-murray) [reply hera_sim/visibilities/cli.py:170]: Well, all other processes exit in the statement above, so only rank 0 does anything beyond that point. I thought it was nicer to not have the extra unnecessary indent.

Comment (piyanatk) [reply hera_sim/visibilities/cli.py:170]: Ah yes, I see now. The rank checking is not necessary then.

**Pull request merged**